sdks/java/io/hadoop/jdk1.8-tests/pom.xml

```diff
@@ -125,7 +125,6 @@
   </profiles>
 
   <properties>
-    <log4j.core.version>2.6.2</log4j.core.version>
     <hadoop.common.version>2.7.0</hadoop.common.version>
     <guava.version>19.0</guava.version>
```

Comment:
looks like opportunities to move this versioning mumbo jumbo into dependency management in the root hadoop pom.xml.